api-streaming/README.md

```diff
@@ -0,0 +1,24 @@
+API Streaming
```

Comment:
not enough info in readme
elaborate with example, setup prerequisites, goal, ...

Reply:
done